README.md

```diff
@@ -1 +1,17 @@
-# napalm-brocade
\ No newline at end of file
+# Napalm Brocade Driver
+
+This driver is vendor specific south-bound implentation of the NAPALM API. Supported here are SLX and NOS based switches.
+
+## Instructions
+
+Please see instuction at:
+https://napalm.readthedocs.io/en/latest/
+
+##Caveats
```

Comment:
This should be send to the `napalm` repo with the rest of the drivers' caveats.

https://github.com/napalm-automation/napalm/tree/develop/docs/support